components/icons/Icons.tsx

```
@@ -126,3 +128,92 @@ export const IconDirectory = () => {
+
+export const IconEverworksSimple = (props?: SVGProps<SVGSVGElement>) => {
+  return (
+    <svg
+      {...props}
+      width="50"
+      height="50"
+      viewBox="0 0 134 134"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className="w-full h-full"
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)